.github/workflows/pull-request-conditionals.yaml

```diff
@@ -24,26 +38,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  lint-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
-      - name: Use Node.js latest
-        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
-        with:
-          node-version: 20
-      - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@master
-      - name: Install UDS CLI
-        shell: bash
-        # renovate: datasource=github-tags depName=defenseunicorns/uds-cli versioning=semver
-        run: brew install defenseunicorns/tap/uds@0.14.2
-      - name: Run Formatting Checks
-        run: uds run lint-check --no-progress
-
   # This job checks if there are changes in specific paths source packages.
   check-paths:
-    needs: lint-check
```

Comment:
Preferably would like to keep this dependent on linting for better fast feedback/cancelling of workflows when something fails lint. As discussed we could pull the lint into a shareable workflow/action to let us call it in both the shim and this workflow.